SR2020/2020-05-26-virtual-competition-details.md

```diff
@@ -0,0 +1,32 @@
+---
+to: sr2020-teams
+subject: Student Robotics 2020 Virtual Competition Details
+---
+
+![](https://pbs.twimg.com/media/EYt9wgcX0AAKjOn?format=jpg)
+
+Hello!
+
+Today's the day we announce some long awaited details of the virtual competition for Student Robotics 2020.
+
+## Simulator
+
+For the virtual competition, we've created a new simulation, based in [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available in the docs.
+
+[Competition Simulator Docs](https://studentrobotics.org/docs/competition-simulator/)
+
+Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator, however we've tried to keep it as similar as possible, and easily adaptable from what you've already written.
+
+## Competition Format
+
+Our intention is to run this in a very similar way to the regular competition. In early July (subject to change), we'll run virtual competition events, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.
```

Comment:
```suggestion
Our intention is to run this in a very similar way to the regular competition. In early July (subject to change), we'll run virtual competition sessions, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.
```

As I understand the plan we're not running a single event as such -- just the sessions as described. I suggest that pick a single term an stick to it.
Alternatively, let's clarify that an event is comprised of several sessions (or whatever the relationship is between the two concepts).